### ISSUE TITLE
Etudiants bloqués dans les murs

### ISSUE THREAD

**Issue by Sterla1** [JulienNigon/SpellcasterUniversity-Bug-Report#1178]
Hello ! 

Déjà merci pour tout le travail que vous faites et toutes les dernières MAJ c'est vraiment un bonheur de jouer à ce jeu ! 

Petit feedback sur le terrain des Moines :
Vers la fin de cette partie, plusieurs étudiants se sont retrouvés bloqués dans le mur comme s'ils cherchaient à se diriger  à droite de la pièce (bibliothèque) où ils étaient. Sauf que la bibliothèque était en étage à la limite de la zone de construction basse du terrain ; on aurait dit qu'ils voulaient se diriger dans les pièces de la zone de construction haute ! 

J'ai un peu observé le comportement des étudiants bloqués, des commentaires continuaient à apparaître comme s'ils n'étaient bloqués que visuellement, les besoins augmentaient sauf que pas d'augmentation d'apprentissage ! 

J'espère que cela pourra aider, bonnes fêtes à vous tous ! :)

<img width="1384" alt="Pasted Graphic" src="https://user-images.githubusercontent.com/76745788/103302410-df800b00-4a03-11eb-9d2c-7a170c50f79b.png">
<img width="1304" alt="Pasted Graphic 1" src="https://user-images.githubusercontent.com/76745788/103302414-e1e26500-4a03-11eb-847e-f4e50f1382a8.png">
<img width="713" alt="Pasted Graphic 2" src="https://user-images.githubusercontent.com/76745788/103302417-e444bf00-4a03-11eb-8c30-cc1b68fcd626.png">

**Comment by Neelan22** [comment]
Bonjour Sterla1,
merci pour ce retour, c'est la première fois qu'on l'a sur cette carte. Par hasard, aurais tu encore la sauvegarde sous la main?

**Comment by Sterla1** [comment]
J'ai peut-être encore la sauvegarde oui mais j'ai fini la campagne associé à l'évènement, ça marche quand même pour vous ? Si oui tu peux m'expliquer comment accéder à la sauvegarde (sur Mac) ? :)

**Comment by Neelan22** [comment]
Non, la partie a été écrasée par le(s) tableau(x) suivant. Merci quand même !

**Comment by Sterla1** [comment]
Oh zut ! Je resterai à l’affût en repassant sur la map alors !

**Comment by Neelan22** [comment]
Je réouvre l'issue parce que le bug reste présent de temps en temps sur cette carte
